docs/gettingstarted/index.rst

```diff
@@ -99,7 +99,7 @@ Getting started with Flocker
             you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
               virtualenv flocker-tutorial && \
               flocker-tutorial/bin/pip install --upgrade pip && \
-              flocker-tutorial/bin/pip install --quiet https://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-|latest-installable|-py2-none-any.whl && source flocker-tutorial/bin/activate
+              flocker-tutorial/bin/pip install --quiet https://s3.amazonaws.com/clusterhq-archive/python/Flocker-|latest-installable|-py2-none-any.whl && source flocker-tutorial/bin/activate
```

Comment:
Testing the equivalent URL on `clusterhq-archive-testing` doesn't appear to work. It looks like https://clusterhq-archive-testing.s3.amazonaws.com/ is needed instead.